superbench/benchmarks/directx_benchmarks/utils/D3D12Timer.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
It's third party as well?

Reply:
no, I have rewritten this class to make it simple and can be used for both compute and copy engine